AgileVentures/LocalSupport
# lock down a specific node version

#### What does this PR do?
- Lock our project to a more recent version of node that generates a package.json
#### Description of Task to be completed?
- Lock our project to a more recent version of node that generates a package.json
#### How should this be manually tested?
- Clone the repo
- Checkout into this branch
- Run `npm-install`
If your node version is less than the one specified you should get an error
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
[148842665](https://www.pivotaltracker.com/n/projects/742821/stories/148842665  )

#### Screenshots (if appropriate)
- N/A
#### Questions: 
- N/A
